harsh-vador/tic-tac-toe/styles.css

```diff
@@ -0,0 +1,237 @@
+html {
+  box-sizing: border-box;
+  text-align: center;
+  margin-top: 50px;
+}
+.submit_button {
```

Comment:
check class naming convention. naming convention should be consistent for all class names

Reply:
Changes done